controllers/drplacementcontrol_controller.go

```diff
@@ -565,8 +593,9 @@ func (r *DRPlacementControlReconciler) SetupWithManager(mgr ctrl.Manager) error
 	r.eventRecorder = rmnutil.NewEventReporter(mgr.GetEventRecorderFor("controller_DRPlacementControl"))
 
 	return ctrl.NewControllerManagedBy(mgr).
-		WithOptions(ctrlcontroller.Options{MaxConcurrentReconciles: getMaxConcurrentReconciles(ctrl.Log)}).
+		WithOptions(ctrlcontroller.Options{MaxConcurrentReconciles: getMaxConcurrentReconciles(r.RmnConfig)}).
 		For(&rmn.DRPlacementControl{}).
+		Watches(&source.Kind{Type: &corev1.ConfigMap{}}, handler.EnqueueRequestsFromMapFunc(r.configMapMapFun)).
```

Comment:
Add a predicate that invokes this only on config map version change, as in [here](https://github.com/RamenDR/ramen/blob/372456d013ec575c20b3094ee67b2e1d1884ca61/controllers/drpolicy_controller.go#L365)